README.md

```diff
@@ -70,6 +70,19 @@ cmake -G "Visual Studio 17 2022 Win64" -B build
 # Generates MSVS solution build/LuaBridge.sln
 ```
 
+## Installing LuaBridge(vcpkg)
+
+You can download and install LuaBridge using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+```Powershell or bash
+git clone https://github.com/Microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.sh
+./vcpkg integrate install
```

Comment:
Should we mention command set for Windows?

Reply:
Of course, this can help users better use. 
It should be `./bootstrap-vcpkg.bat` for `Windows`.